py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -83,6 +84,11 @@ def compile(
     hardware_compatible: bool = _defaults.HARDWARE_COMPATIBLE,
     timing_cache_path: str = _defaults.TIMING_CACHE_PATH,
     lazy_engine_init: bool = _defaults.LAZY_ENGINE_INIT,
+    save_engine_cache: bool = _defaults.SAVE_ENGINE_CACHE,
+    load_engine_cache: bool = _defaults.LOAD_ENGINE_CACHE,
+    engine_cache_dir: str = _defaults.ENGINE_CACHE_DIR,
```

Comment:
Optional in the case of a user cache instance?

Reply:
Maybe we should specialize these settings like `disk_engine_cache_dir` or we should document that if you give engine_cache_instance, then these do nothing